### PR TITLE
ui: display authors native names

### PR DESCRIPTION
**Pull request by monaawi**
* INSPIR-2927

### CODE REVIEW

File: ui/src/authors/components/AuthorResultItem.jsx [hunk 1]
```diff
@@ -21,11 +21,12 @@ class AuthorResultItem extends Component {
     );
     const arxivCategories = metadata.get('arxiv_categories');
     const experiments = metadata.get('project_membership');
+    const nativeName = metadata.getIn(['name', 'native_names', 0]);
```
Comment (michamos) [inline ui/src/authors/components/AuthorResultItem.jsx]: what if there's more than one native name?

Comment (StellaCh) [reply ui/src/authors/components/AuthorResultItem.jsx]: we only allow 1 native name in the author form. Are there cases with more?

Comment (monaawi) [reply ui/src/authors/components/AuthorResultItem.jsx]: the schema has a list of native_names so it allows for more than one to be there. But do we really want to display all native names in the title like that?

Comment (michamos) [reply ui/src/authors/components/AuthorResultItem.jsx]: maybe not in the title, but in the detailed record we probably should.

Comment (monaawi) [reply ui/src/authors/components/AuthorResultItem.jsx]: We checked and found only 6 authors out of 125k with more than one native name. After discussion with Stella, we will display only first native name

File: ui/src/authors/components/AuthorName.jsx [hunk 2]
```diff
@@ -5,15 +5,21 @@ import { getAuthorDisplayName } from '../utils';
...
+        {displayName}
+        {nativeName && <span> ({nativeName})</span>}
+      </span>
+    );
   }
 }
 
 AuthorName.propTypes = {
   name: PropTypes.instanceOf(Map).isRequired,
+  nativeName: PropTypes.string,
```
Comment (harunurhan) [inline ui/src/authors/components/AuthorName.jsx]: `nativeName` is already inside `name` prop, so this isn't needed.

Comment (monaawi) [reply ui/src/authors/components/AuthorName.jsx]: right, missed this

File: ui/src/authors/components/__tests__/AuthorName.test.jsx [hunk 3]
```diff
@@ -11,6 +11,15 @@ describe('AuthorName', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('renders with native_name', () => {
+    const name = fromJS({
+      preferred_name: 'Harun Urhan',
+      native_names: ['赵新丽'],
```
Comment (harunurhan) [inline ui/src/authors/components/__tests__/AuthorName.test.jsx]: I think my native name is misspelled little bit 😄

**Pull request merged**